frontend/src/pages/index.js

```diff
@@ -2,12 +2,17 @@ import { useKeycloak } from '@react-keycloak/web';
 import React from 'react';
 import EligibilityPageContainer from '../components/Eligibility/EligibilityPageContainer';
 import Layout from '../components/Layout';
+import {Helmet} from "react-helmet";
 
 function Index() {
   const { keycloak } = useKeycloak();
 
   return (
     <div className="app">
+       <Helmet>
+            <meta charSet="utf-8" />
```

Comment:
This meta tag is already in `public/index.html` so you don't need to add it again here.